packages/playground/src/utils/deploy_helpers.ts

```diff
@@ -11,6 +11,30 @@ export function createNetwork(network: Network = {}): NetworkModel {
   return nw;
 }
 
+export function setCaproverWorkers(vms: ZmachineData[], projectName: string | undefined = undefined): ZmachineData {
```

Comment:
Why do we expect a projectName while we're setting caprover workers?

Reply:
Won't the project in this case always be Caprover?

Reply:
> Why do we expect a projectName while we're setting caprover workers?

if i understand well, we save the workers as the leader name to be able to load them, so why don't I hard-code it? I don't want to set it in all of the calls. I want to set it only after deploying a new worker